build-config/Makefile

```diff
@@ -162,7 +162,7 @@ ifneq (,$(MAKECMDGOALS))
       $(error Unable to find valid machine configuration directory.)
     endif
     include $(MACHINEDIR)/machine.make
-    ARCHDIR ?= $(realpath ./arch)
+    ARCHDIR ?= $(realpath ./arch/$(ONIE_ARCH))
```

Comment:
This looks correct; however, when attempting to build the ```kvm_x86_64``` platform (the default) I'm receiving an error.

A simple way to reproduce the error is:
```
$ make download
Makefile:166: /x86_64.make: No such file or directory
make: *** No rule to make target '/x86_64.make'.  Stop.
```

The same error appears when specifying the platform:
```
$ make MACHINE=kvm_x86_64 download
Makefile:166: /x86_64.make: No such file or directory
make: *** No rule to make target '/x86_64.make'.  Stop.
```